app/server/data-loaders/tag-page-data.js

```diff
@@ -1,13 +1,29 @@
+import { get } from "lodash";
 import { Story } from "@quintype/framework/server/api-client";
 import { storyToCacheKey } from "@quintype/framework/server/caching";
+import { TAG_PAGE_URL_PREFIX } from "../../isomorphic/constants";
 
 export function loadTagPageData(client, tagSlug, config) {
   return Story.getStories(client, "top", { tag: tagSlug, limit: "20" }).then(
-    stories => ({
-      stories: stories.map(story => story.asJson()),
-      cacheKeys: stories.map(story =>
-        storyToCacheKey(config["publisher-id"], story)
-      )
-    })
+    stories => {
+      return client.getTags(tagSlug).then(({ tags }) => {
+        const tagName = get(tags, [0, "name"], tagSlug);
```

Comment:
We should move client to a wrapper. We could do this later. Add a comment saying `TODO: replace client with Tag wrapper`